<ui version="4.0" >
 <class>BuildingDialog</class>
 <widget class="QDialog" name="BuildingDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Building Parameters</string>
  </property>
  <widget class="QDialogButtonBox" name="okCancelButtons" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>260</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="_buildingGroup" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>10</y>
     <width>191</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title" >
    <string>Building Parameters</string>
   </property>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>80</y>
      <width>53</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Length</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="_widthField" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>110</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="_heightField" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>140</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>110</y>
      <width>53</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Width</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="_buildingExists" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>Is there a Building?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>140</y>
      <width>53</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Depth</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lengthField" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>80</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="_volume" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>170</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>170</y>
      <width>53</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Volume</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelButtons</sender>
   <signal>accepted()</signal>
   <receiver>BuildingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelButtons</sender>
   <signal>rejected()</signal>
   <receiver>BuildingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
